hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -140,7 +140,7 @@ public ContainerBalancerConfiguration(OzoneConfiguration config) {
     // greater than container size
     long size = (long) ozoneConfiguration.getStorageSize(
         ScmConfigKeys.OZONE_SCM_CONTAINER_SIZE,
-        ScmConfigKeys.OZONE_SCM_CONTAINER_SIZE_DEFAULT, StorageUnit.GB) +
+        ScmConfigKeys.OZONE_SCM_CONTAINER_SIZE_DEFAULT, StorageUnit.BYTES) +
```

Comment:
@siddhantsangwan Another issue is the StorageUnit here.
The value from Unit.GB is "5 + 1GB", need to changed to Unit.BYTES.

Reply:
Yes, I was just going to raise a jira for this particular bug when I saw that you've pushed an update. Thanks!

Reply:
Seems the other configs, like "move.timeout" and "balancing.interval", also have the same issue.